src/System.Management.Automation/engine/LanguagePrimitives.cs

```diff
@@ -5006,13 +5006,11 @@ internal static object ThrowInvalidConversionException(object valueToConvert, Ty
                 return CacheConversion<StringCollection>(fromType, toType, LanguagePrimitives.ConvertToStringCollection, rank);
```

Comment:
@iSazonov, your last commit had 1 failures in `PowerShell-CI-macos`
Validate start of console host.No new assemblies are loaded
```powershell
Expected exactly $null, but got @{InputObject=System.Configuration.ConfigurationManager.dll; SideIndicator==>}.
at <ScriptBlock>, /Users/runner/runners/2.160.0/work/1/s/test/powershell/Host/Startup.Tests.ps1: line 125
125:         $diffs | Should -BeExactly $null
```